seldon-core-operator/rockcraft.yaml

```diff
@@ -0,0 +1,115 @@
+name: seldon-core-operator
+summary: Convert your ML models or language wrappers into microservices
+description: |
+  Seldon core converts your ML models (Tensorflow, Pytorch, H2o, etc.) or
+  language wrappers (Python, Java, etc.) into production REST/GRPC
+  microservices.
+
+  Seldon handles scaling to thousands of production machine learning models and
+  provides advanced machine learning capabilities out of the box including
+  Advanced Metrics, Request Logging, Explainers, Outlier Detectors, A/B Tests,
+  Canaries and more.
+
+version: v1.15.1_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  manager:
+    override: replace
+    summary: "seldon-core service"
+    startup: enabled
+    user: ubuntu
+    command: "/opt/seldon-core-operator/manager --enable-leader-election --webhook-port 4443 --create-resources true"
+platforms:
+  amd64:
+
+parts:
+  seldon-core-operator:
+    plugin: go
+    source: https://github.com/SeldonIO/seldon-core
+    source-type: git
+    source-tag: v1.15.1
+    build-snaps:
+      - go
+      - kustomize/latest/stable
+    override-build: |
+      set -xe
+      cd $CRAFT_PART_SRC/operator
+
+      # Download go dependencies and build the manager binary
+      rm -f go.sum
+      go mod tidy -compat=1.17
+      go mod download all
+      go build -a -o manager main.go
+
+      # Install the manager binary and the licenses file
+      install -D manager $CRAFT_PART_INSTALL/opt/seldon-core-operator/manager
+      install -D licenses/license.txt $CRAFT_PART_INSTALL/opt/seldon-core-operator/licenses/license.txt
+
+      # TODO: Validate if we indeed need to do this given that the charm overrides the /tmp dir
+      # Generate the resources per the upstream OCI image
+      make generate-resources
+
+      # Install the generated resources into /tmp/operator-resources
+      res="$CRAFT_PART_INSTALL/tmp/operator-resources/"
+      install -D generated/v1_service_seldon-webhook-service.yaml "${res}/service.yaml"
+      install -D generated/v1_configmap_seldon-config.yaml "${res}/configmap.yaml"
+      install -D \
+        generated/admissionregistration.k8s.io_v1_validatingwebhookconfiguration_seldon-validating-webhook-configuration.yaml \
+        "${res}/validate.yaml"
+      install -D \
+        generated/apiextensions.k8s.io_v1_customresourcedefinition_seldondeployments.machinelearning.seldon.io.yaml \
+        "${res}/crd-v1.yaml"
+
+      # security requirement
```

Comment:
Maybe we want to put it into separate part 

```
security-team-requirement:
    plugin: nil
    override-build: |
      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
```

Reply:
Ok. Will do. We will need to address it in other rocks as well.